DB/projects.json

```diff
@@ -219,11 +212,8 @@
   {
     "github_username": "adity2911",
     "Social_media": {
-      "gitHub": "https://github.com/adity2911",
       "LinkedIn": "https://www.linkedin.com/in/aditya-tulsiyan-585b50111/",
-      "Instagram": "YOUR_INSTAGRAM_ACCOUNT_LINK",
-      "YouTube": "YOUR_YOUTUBE_ACCOUNT_LINK",
-      "Twitter": "YOUR_TWITTER_ACCOUNT_LINK"
+      "Twitter": ""
```

Comment:
what about blank social link? can we avoid it?

Reply:
Yaa, we can avoid them. Let me do that.

Reply:
Hey @priyankarpal, I have removed the blank social link. Kindly review it now.